src/components/Expandable.tsx

```diff
@@ -22,7 +22,7 @@ export default function Expandable({ title, description, expanded, setExpanded }
 					<UpArrowIcon />
 				</ExpandableTrailingIcon>
 			</ExpandableTitle>{' '}
-			{expanded && <p>{description}</p>}
+			{expanded && <>{description}</>}
```

Comment:
What's the with empty tags?

Reply:
It is a React trick to express html without using a real element. There were errors being thrown in the console here for nesting `<p>` tags inside of other `<p>` tags, so this was the fix.

Often when creating HTML in React, the compiler will complain about using more than one parent element:

```tsx
return(
  <div>My div</div>
  <div>My other div</div>
  ) // ERROR: React component must return one parent
```

You can use an empty tag to remedy these situations without rendering a real element on the DOM:

```tsx
return(
  <>
    <div>My div</div>
    <div>My other div</div>
  </>
  ) // This is fine
```

Reply:
:the more you know: